init.sh

```diff
@@ -31,12 +31,15 @@ cp -R "$temp_directory"/* "$temp_directory"/.??* "$install_directory"
 
 # Move workflow files and configuration
 if [ "$install_directory" != "." ]; then
-    echo "Moving workflow files and configuration..."       
+    echo "Moving workflow files and configuration..."
     if [ ! -d .github/workflows ]; then
         mkdir -p .github/workflows
     fi
     mv "$install_directory/.github/workflows"/* .github/workflows
     mv "$install_directory/.github/.jira_sync_config.yaml" .github/
+    if [ ! -f .wokeignore ]; then
```

Comment:
If we're entertaining the idea that `.wokeignore` may exist in the root we should follow up with a message (echoed to STDOUT) to note the existence of such a file. Otherwise we're admitting that having two possibly different but identically-named files is okay.

We may consider renaming an existing file and creating the symlink.

Reply:
Good point, added.
I don't think we should rename the existing file though - that might break other things for the user. And it's easier to debug something that you are currently trying to set up than figuring out why something that used to work suddenly stopped.